### PR TITLE
raine: Update to version 0.95

### PR DESCRIPTION
**Pull request by RavenMacDaddy**
Also reorders Autoupdate section to have the same order as the installation section.

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RavenMacDaddy** [comment]
Does anyone see the culprit?

**Comment by RavenMacDaddy** [comment]
I really don't know any more...no errors in IDE, formatted with tools in NP++, Autoupdate is now LITERALLY the same as before the commit, but it doesn't work.

Insight is greatly appreciated.

**Comment by RavenMacDaddy** [comment]
…and now changes become empty? What's actually going on, did the hub just break on itself? I'll gladly be wrong about everything, I just don't see it.

File: bucket/raine.json [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
+		"demos",
+		"emudx",
+		"roms",
+		"savedata",
+		"savegame",
+		"screens"
+	],
+	"checkver": {
+		"url": "https://raine.1emulation.com/download/latest.html",
+		"regex": "Available files for version ([\\d.]+[a-zA-Z]?)"
```
Comment (kazzarin) [inline bucket/raine.json:68]: checkver is reading `0.95.0` from the site, while the binaries use `0.95`
```suggestion
		"regex": "/archive/raine32-([\\d.]+[a-zA-Z]?)\\.7z"
```

**Review by kazzarin — COMMENTED** [review]
The current tests are failing because it prefers spaces over tabs.

**Comment by RavenMacDaddy** [comment]
> The current tests are failing because it prefers spaces over tabs.

Thank you, I'll get right to it. Also, appreciate the regex fix @kazzarin.

**Comment by RavenMacDaddy** [comment]
Did I read right that the test is actually failing on the `racelab` manifest, rather than this commit?

**Comment by Calinou** [comment]
> Did I read right that the test is actually failing on the `racelab` manifest, rather than this commit?

Yes; this should be fixed after rebasing this PR now that I merged https://github.com/Calinou/scoop-games/pull/833.

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**